meta-resin-common/recipes-connectivity/networkmanager/networkmanager_%.bbappend

```diff
@@ -32,6 +36,10 @@ do_install_append() {
     fi
 
     ln -s /var/run/resolvconf/interface/NetworkManager ${D}/etc/resolv.dnsmasq
+
+    # remove these empty not-used (at this moment) directories so we don't have to package them
+    rm -rf ${D}${libdir}/NetworkManager/conf.d
+    rm -rf ${D}${libdir}/NetworkManager/VPN
```

Comment:
If we used "rmdir" here instead of "rm -rf" then the build would crash if these directories ever become non-empty, or cease to exist.